README.md

```diff
@@ -237,27 +199,24 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Khaled-AlKhateeb/Budget_Capstone/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project please help it stand out by giving it a star ✨
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+- I would like to thank Microvers for providing this great opportunity and for the project guide as well 👏
+- Huge appriciation to [Gregoire Vella](https://www.behance.net/gregoirevella) for providing design system guidelines since this project is built upon that ✨ 
```

Comment:
Hi @Khaled-AlKhateeb 
Please permit me to use this line to request a change on this file.

- [ ] Please fix the link to your `LICENSE` file is broken 
[README GUIDE](https://github.com/microverseinc/readme-template)